app/assets/javascripts/admin/edition_publishing.js

```diff
@@ -217,3 +217,21 @@ jQuery(function($) {
 
   SetTopicsFromPolicy.init();
 }(jQuery));
+
+(function($) {
+  var $policyInput = $('#edition_policy_content_ids'),
+      $importantCheckbox = $('#edition_important');
+
+  var toggleCheckboxIfPoliciesSelected = function() {
+    if ($importantCheckbox >= 1) {
```

Comment:
Not sure what this is trying to check. Should this be `$policyInput.val().length > 0` which would make sure that an element is selected in the policy input?